libs/hdf-converters/src/converters-from-hdf/asff/transformers.ts

```diff
@@ -424,6 +427,31 @@ function getFilename(options?: IOptions): string {
   return slashSplit?.split('/')[slashSplit.split('/').length - 1] ?? '';
 }
 
+//Split and push a string that is above ASFF attribute character limit
+function pushSplitString(
+  pushedStr: string,
+  charLimit: number,
+  arrToPush: string[],
+  fieldName: string
+) {
+  //Account for common escape characters in upper character limit
+  charLimit -= (pushedStr.match(/\"/g) || []).length;
+  charLimit -= (pushedStr.match(/\'/g) || []).length;
+  charLimit -= (pushedStr.match(/\n/g) || []).length;
+  charLimit -= (pushedStr.match(/\t/g) || []).length;
+  charLimit -= (pushedStr.match(/\r/g) || []).length;
+  const cntMax = Math.ceil(pushedStr.length / charLimit);
+  let cntMin = 1;
+  while (pushedStr.length > charLimit) {
+    arrToPush.push(
+      `${fieldName}${cntMin}of${cntMax}/${pushedStr.slice(0, charLimit)}`
+    );
+    pushedStr = pushedStr.slice(charLimit);
+    cntMin++;
+  }
+  arrToPush.push(`${fieldName}${cntMin}of${cntMax}/${pushedStr}`);
```

Comment:
On further reflection, it seems like it's necessary to do the X of Y naming scheme so as to have the asff->hdf mapper work with how we're transforming the types array into an object (and thusly need unique keys, which wouldn't have happened with passthrough being duplicated a whole bunch) (...but maybe you could change that function so that way it creates an array if it sees the same key?  don't do it unless you wanna).

Please do add an Execution/warning (not passthrough_warning) item to the types array if we're gonna have data loss (i.e. the slice of 0,50 actually causes the array to have items removed from it).